src/Components/HandleUserChoice.js

```diff
@@ -0,0 +1,54 @@
+
+import React from 'react';
+
+const HandleUserChoice = (props) => {
```

Comment:
HandleUserChoice appears to represent a function name. Could you please provide a more descriptive name for it, considering it should represent a React component?